phpstan.neon.dist

```diff
@@ -13,11 +13,5 @@ parameters:
         message: "#While loop condition is always true#"
         path: "lib/Commands/BuildWebsiteCommand.php"
         count: 1
-      - # Temporary final classes in tests are covered by nunomaduro/mock-final-classes until the refactorings start
-        message: "#has unresolvable native type#"
-        path: "tests/"
-      - # Temporary final classes in tests are covered by nunomaduro/mock-final-classes until the refactorings start
-        message: "#contains unresolvable type#"
-        path: "tests/"
```

Comment:
Looks like you can remove the nunomaduro/mock-final-classes dev dependency from composer.json now, right? :tada: